src/js/angular/controllers.js

```diff
@@ -461,12 +461,53 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
             // if it's free access check if we still can access the current repo
             // if not, a new default repo will be set or the current repo will be unset
             $repositories.resetActiveRepository();
+            $rootScope.updateReturnUrl();
         } else if ($jwtAuth.isSecurityEnabled()) {
             // otherwise show login screen if security is on
             $rootScope.redirectToLogin();
         }
     };
 
+    $scope.showMainManuAndStatusBar = () => {
+        return $jwtAuth.isAuthenticated() || $scope.isSecurityEnabled() && $scope.isFreeAccessEnabled();
+    };
+
+    let isAuthenticated = undefined;
```

Comment:
Would there be a problem if this is initialized with `false`?

Reply:
In the initial implementation, it had to be undefined, but now it can be false. I will verify and set it.

Reply:
Actually, it has to be undefined, I wrote an explanatory comment to describe the initialization of the variable.